tests/core/pyspec/eth2spec/test/altair/unittests/validator/test_validator.py

```diff
@@ -85,12 +85,9 @@ def _get_sync_committee_signature(
     pubkey = state.current_sync_committee.pubkeys[sync_committee_index]
     privkey = pubkey_to_privkey[pubkey]
 
-    domain = spec.get_domain(
-        state,
-        spec.DOMAIN_SYNC_COMMITTEE,
+    return compute_sync_committee_signature(
+        spec, state, target_slot, privkey, block_root=target_block_root
     )
-    signing_data = spec.compute_signing_root(target_block_root, domain)
-    return bls.Sign(privkey, spec.hash_tree_root(signing_data))
```

Comment:
looks like there was a bug here anyway! should not be taking a second `hash_tree_root` and this likely worked only because the root of a root is just itself...